spec/catchers/googlebooks_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'GoogleBooks' do
+
+  context 'catch a book by' do
+    it 'isbn' do
+      book = GoogleBooks.search('isbn:9781118084786').first
+      expect(book.title).to eq 'Ruby on Rails For Dummies'
+    end
+
+    it 'title' do
+      search = GoogleBooks.search('intitle:Darth Vader')
+      titles = search.map(&:title) unless search.total_items.zero?
```

Comment:
@rafaelrpbelo why this `unless search.total_items.zero?`?

I think it can be just:

```
it 'title' do
  books = GoogleBooks.search('intitle:Darth Vader').map(&:title)

```